test/utils/cryptography.js

```diff
@@ -0,0 +1,286 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import crypto from '../../src/utils/cryptography';
+// Required for stubbing
+const elements = require('lisk-elements');
+
+describe('crypto utils', () => {
+	describe('elements throws error', () => {
+		const errorMessage = 'some error';
+		beforeEach(() => {
+			return sandbox
+				.stub(elements.default.cryptography, 'encryptMessageWithPassphrase')
```

Comment:
As mentioned in the previous review, it should be possible to stub `elements.cryptography` and make these stubbing statements slightly more succinct/readable.